src/PKSim.Core/Services/TransportContainerUpdater.cs

```diff
@@ -58,9 +59,27 @@ public void SetDefaultSettingsForTransporter(ISimulationSubject simulationSubjec
          foreach (var transporterContainer in simulationSubject.AllMoleculeContainersFor<TransporterExpressionContainer>(transporter))
          {
             updateTransporterContainerFromTemplate(transporterContainer, null, transportType);
+            updateFractionExpressedEpithelial(transporterContainer, transportType);
          }
       }
 
+      private void updateFractionExpressedEpithelial(TransporterExpressionContainer transporterContainer, TransportType transportType)
+      {
+         if (!transportType.IsOneOf(TransportType.Efflux, TransportType.Influx))
+            return;
+         
+         var fractionExpressedEpithelial = transporterContainer.Parameter(CoreConstants.Parameters.FRACTION_EXPRESSED_EPITHELIAL);
+         if (fractionExpressedEpithelial == null)
+            return;
+
+         //value was set by the user.
+         if (fractionExpressedEpithelial.Value != 0 && fractionExpressedEpithelial.Value != 1)
+            return;
+         
+         //Set the value according to the new transport type
+         fractionExpressedEpithelial.Value = transportType == TransportType.Efflux ? 1 : 0;
```

Comment:
is this function called only for "Organs with Lumen" (Liver, Kidney, GI) or for all organs? Because for other organs
fraction_expressed_Epithelial should be always const=0

Reply:
fractionExpressedEpithelial only exists for organ with lumen by construction

Reply:
here
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Core/Services/IndividualTransporterFactory.cs#L145

Reply:
ok, all good then